lib/operations/topics/publish.js

```diff
@@ -10,17 +10,17 @@ const encodingStrategies = {
 
 const getBodyEncoded = (body, contentEncoding) => (encodingStrategies[contentEncoding] || encodingStrategies.default)(body);
 
-module.exports = sender => async (body, { messageId, label = '', contentType, contentEncoding, applicationProperties, scheduledEnqueueTimeUtc } = {}) => { // eslint-disable-line object-curly-newline
+module.exports = sender => async (body, options = {}) => { // eslint-disable-line object-curly-newline
+	const { label = '', contentEncoding, applicationProperties, scheduledEnqueueTimeUtc, ...messageFields } = options
 	const message = {
-		body: getBodyEncoded(body, contentEncoding),
-		messageId,
 		label,
-		contentType,
 		applicationProperties: {
 			contentEncoding,
-			...applicationProperties,
 			attemptCount: 0,
+			...applicationProperties,
 		},
+		...messageFields,
```

Comment:
Apart from the `messageId`, are there any other important properties?

Reply:
We should add the `deliveryCount` as well, but maybe for next PR

Reply:
I think `deliveryCount` will be important for us

Reply:
That's on `applicationProperties` if I'm not mistaken

Reply:
Looking at the `ServiceBusMessage` interface, I see a lot of fields people might want to use:
- contentType,
- partitionKey,
- sessionId
- replyToSessionId
- timeToLive
WE don't need them now, but someone else might. I don't see any good reason not to make these other fields available.